packages/release/repart-local.service

```diff
@@ -13,7 +13,7 @@ RequiresMountsFor=/local
 Type=oneshot
 
 # Resize the partition, whether or not it resides on the same disk as /.
-ExecStart=/usr/bin/systemd-repart --dry-run=no /dev/disk/by-partlabel/BOTTLEROCKET-DATA
+ExecStart=-/usr/bin/systemd-repart --dry-run=no /dev/disk/by-partlabel/BOTTLEROCKET-DATA
```

Comment:
Doesn't the dash mean that this unit will always be marked as successful? Is this change necessary? If the repart fails then this is marked as success, what happens?

Reply:
Yes, that's intentional. In almost all cases this repart call will be no-op and do nothing since the prior repart call in `label-data-preferred` will have set up the preferred data partition already. 

In cases where the preferred data partition does not exist and only the alternative data partition is viable, since `release-repart-local.conf` matches against the preferred data partition type GUID, this call will always fail. But that's OK since the repart would have been handled by `label-data-alternative` and we can directly `growfs` the `/local` fs. If both preferred/alternative data partitions are not available/viable, then this service will never run since `local.mount` would have failed.

The reason why we still keep the `systemd-repart` call here still is for backward compatibility purposes. It is reasonable for someone who used Bottlerocket before this change to expect that as long as they attach a volume/disk for a partition labelled `BOTTLEROCKET_DATA` and with the correct partition type GUID, the host will repart that partition to grow it to max disk size. We do not want to break that workflow.